config/initializers/highlight_override.rb

```diff
@@ -117,3 +117,23 @@ def update_model!(model)
     model.tap(&:save!)
   end
 end
+
+Api::V0::Bindings::GetHighlightsSummaryParameters.class_exec do
+  def summarize(user_uuid:)
+    highlights = ::Highlight.by_user(user_uuid)
```

Comment:
kinda feels like this class (or a service class) should do this stuff, and have unit tests wrapped around it for all the filtering variations and grouping.  and then the controller can be kept golden path.

Reply:
We don’t have that for other examples in this file do we?